CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ set(CMAKE_BUILD_TYPE Release)
 file(GLOB TAG_FILES ${CMAKE_SOURCE_DIR}/tag*.c)
 set_source_files_properties(SOURCE ${TAG_FILES} PROPERTIES COMPILE_FLAGS -O0)
 add_library(${PROJECT_NAME} SHARED ${APRILTAG_SRCS} ${COMMON_SRC} ${TAG_FILES})
+add_library(${PROJECT_NAME}_static STATIC ${APRILTAG_SRCS} ${COMMON_SRC} ${TAG_FILES})
```

Comment:
it would be better to avoid this and remove `SHARED` from the orginal code and let the standard variable `BUILD_SHARED_LIBS` to drive the build for shared or static version

Reply:
This is already proposed in https://github.com/AprilRobotics/apriltag/pull/172.

Reply:
The proposed change has been merged. You can use `BUILD_SHARED_LIBS` now to control the type of library.